src/app/shared/breadcrumps/breadcrumps.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-breadcrumps',
+  templateUrl: './breadcrumps.component.html',
+  styleUrls: ['./breadcrumps.component.scss']
+})
+export class BreadcrumpsComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```

Comment:
no need to declare here empty function and constructor if it not in use. Same for CartComponent